data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,62 @@ mission "Arachi Cartoonist 2"
 				accept
 			label noship
 			`	"Ah, I see. Well, just a reference it would be, so the name matters not, I suppose. Anyway, thanks again Captain."`
+
+
+
+mission "Coalition Folklore: Arach"
+	minor
+	invisible
+	to offer
+		random < 30
+	source
+		attributes "arach" "folklore"
+	on offer
+		conversation
+			`Shortly after entering the spaceport, you observe an Arach in what you guess is a formal "dress," a delicate silk wrap covering most of her body with each leg covered by its own embroidered skirt. She exclaims something loudly into a personal communication device before putting it away with a shove. Like everyone else who has momentarily been witness to this, you go to look away, but the Arach has cast their gaze around, seen you, and immediately begun to approach. "Our honorable human visitor, I salute," she says loudly as she closes the gap. "Say, Captain, appreciate the fine arts, do you? To a cultural soiree, I'm headed, but busy with work, dearest Papa is, so vacant, his reserved seat is. Accompany me, would you?"`
```

Comment:
"soirée" is normally spelled with a "é", which wouldn't be displayed in game. But I can't come up with a good alternative.